src/vs/editor/browser/controller/textAreaHandler.ts

```diff
@@ -553,7 +553,7 @@ export class TextAreaHandler extends ViewPart {
 	private _getAriaLabel(options: IComputedEditorOptions): string {
 		const accessibilitySupport = options.get(EditorOption.accessibilitySupport);
 		if (accessibilitySupport === AccessibilitySupport.Disabled) {
-			return nls.localize('accessibilityOffAriaLabel', "The editor is not accessible at this time. Press {0} for options.", platform.isLinux ? 'Shift+Alt+F1' : 'Alt+F1');
```

Comment:
You need to pull the actual keybindings to reduce repetition and they could potentially be customized. We should also be using formatting it with the label service to ensure consistency.

Reply:
this class is pretty bare / lacks a service injection which is why I was doing this, but makes sense

Reply:
re the label service, i'm not sure which one would be applied here - it's not a workspace, uri, or a host